### PR TITLE
roll: add page

### PR DESCRIPTION
**Pull request by AnimiVulpis**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/roll.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# roll
+
+> Rolls a user-defined dice sequence.
+
+- Roll 3 6-sided dices and sums the results:
```
Comment (sbrl) [inline pages/common/roll.md]: Isn't the singular of dice "die"?

Comment (AnimiVulpis) [reply pages/common/roll.md]: Checked back on this and you are right:
One **die**
Multiple **dice**
**dices** is a verb if I understood correctly (again something learned ^^)

Will change accordingly

**Comment by AnimiVulpis** [comment]
Did I tokenize correctly?

**Comment by agnivade** [comment]
You have to put the {{ }} only once. So something like {{3d}}, {{4d6h3}}.

File: pages/common/roll.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+`roll {{4d6h3}}`
+
+- Roll 2 12-sided dice 2 times and show every roll:
+
+`roll --verbose {{2{2d12}}}`
+
+- Roll 2 20-sided dice until the result is bigger than 10:
+
+`roll "{{2d20>10}}"`
```
Comment (agnivade) [inline pages/common/roll.md]: Just curious why is this example in quotes. Others don't have quotes.

Comment (AnimiVulpis) [reply pages/common/roll.md]: It's necessary because otherwise the output of `roll 2d20` would be piped into a file called `10`

**Comment by sbrl** [comment]
Thanks, @AnimiVulpis! That's another useful command added :smiley_cat:

**Pull request merged**